# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jsaw/e16keyedit/e16-location.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- e16keyedit-0.2/Makefile.orig-rock-maint	2003-08-27 18:10:51.000000000 +0200
+++ e16keyedit-0.2/Makefile	2003-08-29 06:32:11.000000000 +0200
@@ -10,23 +10,23 @@
 BUILD    = \
 
 LIB_DIRS = \
-    -L/usr/local/lib -L/usr/X11R6/lib -L/usr/X11/lib -L/usr/dt/lib
+    -L$(E16)/lib -L/usr/X11R6/lib -L/usr/X11/lib -L/usr/dt/lib
 LIBS     = \
-    `gtk-config --libs` -lgdbm -lgdk_imlib
-CFLAGS   = \
-    -O -g -D_GNU_SOURCE $(DEVFLAGS)
+    `$(GTK_CONFIG) --libs` -lgdbm -lgdk_imlib
+EXTRA_CFLAGS   = \
+    -D_GNU_SOURCE
 #CFLAGS   = \
 #    -O -g -W -Wall -Wno-unused  -pedantic -pedantic-errors -ansi -D_GNU_SOURCE
 OPTFLAGS   = \
     -O6 -mpentium -mcpu=pentium -march=pentium -ffast-math -D_GNU_SOURCE
 INCLUDES = \
-    `gtk-config --cflags` -I..
+    `$(GTK_CONFIG) --cflags` -I..
 
 BIN      = e16keyedit
 
 # configurable stuff
-BINARY_DESTINATION = /usr/local/enlightenment/bin
-DATA_DESTINATION   = /usr/local/enlightenment/share
+BINARY_DESTINATION = $(E16)/bin
+DATA_DESTINATION   = $(E16)/share
 
 # commands to do things
 INSTALL  = install
@@ -92,14 +92,8 @@
 	$(RMRF) $(BIN) *.CKP *.ln *.BAK *.bak *.o core errs ,* *~ *.a .emacs_* tags TAGS make.log MakeOut $(DIST) $(DIST).tar $(DIST).tar.gz ./=build ./=install "#"*
 
 # how to install
-install:: e
-	@echo 
-	@echo 
-	@echo "You don't really want to do this. it doesn't work yet."
-	@echo 
-	@echo 
-#	for I in $(BIN); do $(INSTALL) -D -m 0775 $$I $(prefix)$(BINARY_DESTINATION)/$$I; done
-#	for I in $(DATA); do $(INSTALL) -D -m 0664 $$I $(prefix)$(DATA_DESTINATION)/$$I; done
+install:: 
+	for I in $(BIN); do $(INSTALL) -D -m 0775 $$I $(BINARY_DESTINATION)/$$I; done
 
 # how to make a dist tarball
 dist::
